File: app/src/main/java/com/example/igreeldownloader/models/FacebookReel.java
package com.example.igreeldownloader.models;

import java.util.ArrayList;

public class FacebookReel {
    public boolean error;
    public String msg;
    public ArrayList<Data> data;

    public static class Data {
        public String url;
        public String format;
        public String ext;
        public String format_id;
    }

}

File: app/src/main/java/com/bluelock/realdownloader/models/FacebookReel.kt
package com.bluelock.realdownloader.models

class FacebookReel {
    var error = false
    var msg: String? = null
    var data: ArrayList<Data>? = null

    class Data {
        var url: String? = null
        var format: String? = null
        var ext: String? = null
        var format_id: String? = null
    }
}